lib/applications/resolvers.ts

```diff
@@ -1064,9 +1066,20 @@ export const updateApplicationPaymentInformation: Resolver<
   MutationUpdateApplicationPaymentInformationArgs,
   UpdateApplicationPaymentInformationResult
 > = async (_parent, args, { prisma }) => {
-  // TODO: Validation
   const { input } = args;
-  const { id, donationAmount, shippingPostalCode, billingPostalCode, ...data } = input;
+
+  try {
+    await paymentInformationMutationSchema.validate(input);
+  } catch (err) {
+    if (err instanceof ValidationError) {
+      return {
+        ok: false,
+        error: err.message,
```

Comment:
Double check if we need to also have `error: null` in the response object on success

Reply:
Good catch, yeah i think you're right that its needed